CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # @digitalbazaar/oauth2-token-endpoint-handler ChangeLog
 
+## 2.0.0 - 2021-xx-xx
+
+### Added
+- **BREAKING**: Added new parameter `cache` to `tokenExchangeHandler`.
```

Comment:
We should also say that using a cache is important for security purposes because it limits the number of access tokens issued to each client.

Reply:
Updated.